JSDemos/Demos/DataGrid/GridSummaries/description.md

```diff
@@ -1 +1,11 @@
-The DataGrid component allows you to&nbsp;display data summaries within its container. This example illustrates the use of&nbsp;total summaries. The demo is&nbsp;configured using the **totalItems** array declared within the **summary** object. This array contains several objects that specify summary settings for each item.
\ No newline at end of file
+A total summary creates a synopsis for all data of a certain column. To configure this feature, add the [summary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/) object, instatiate the **summary**.[totalItems](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/) array and add summary objects. Each object must consist of at least the [column](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#column) and [summaryType](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/totalItems/#summaryType) properties. The former specifies the column that supplies data, the latter sets the aggregate function that applies to this data.
+
+This demo has summaries that do the following:
+
+1. Count the number of orders (summaryType of *"count"*)
```

Comment:
- Ordered list is good for series of connected steps. In our case, list items are independent of each other.
- If you can, describe all in one place, don't refer back to add to the previous description, as you do with `customizeText` and `valueFormat`.

```suggestion
- **Total number of orders**             
This summary uses the *"count"* **summaryType**.
```